### ISSUE TITLE
Wearing Draconic or Wyvern leggings cause the players view to zoom in.

### ISSUE THREAD

**Issue by andrewt1738** [brandon3055/Draconic-Evolution#356]
## Description: 
When wearing wyvern or draconic leggings and you set the speed to a number above zero **IT LOWERS  THE POINT OF VIEW**. It is not supposed to interfere with your point of view but instead it sets the point of view lower than normal which makes the game unplayable. AND NO THIS IS NOT A NORMAL MINECRAFT MECHANIC. For me Draconic-Evolution-1.7.10-1.0.2f seems to work fine
## Base information
* Minecraft version: 1.7.10
* Minecraft Forge version: release 1.7.10-Forge10.13.4.1614-1.7.10
* Draconic Evolution version: Draconic-Evolution-1.7.10-1.0.2h and Draconic-Evolution-1.7.10-1.0.2g and Draconic-Evolution-1.7.10-1.0.2f

**Comment by GreatOrator** [comment]
Do you have a video or pic of this?  I have tested locally and it is working as it should for me.

**Comment by andrewt1738** [comment]
I was partially wrong, in Draconic-Evolution-1.7.10-1.0e it didn't affect the players POV no matter how high the speed was or how low it was. I will get you proof and a mod list by tomorrow as I am away from my computer.

**Comment by GreatOrator** [comment]
A streamer was able to show me what you are talking about in the technic pack.  We are currently trying to narrow down the cause since it does not seem to happen in the pack I use to test.

A mod list will certainly help

**Comment by andrewt1738** [comment]
Give me a few hours



On Jul 26, 2016, at 6:44 AM, GreatOrator <notifications@github.com<mailto:notifications@github.com>> wrote:


A streamer was able to show me what you are talking about in the technic pack. We are currently trying to narrow down the cause since it does not seem to happen in the pack I use to test.

A mod list will certainly help

-

**Comment by andrewt1738** [comment]
Would a download of all mods work?

**Comment by andrewt1738** [comment]
[https://1drv.ms/u/s!ApqqX0JZjRs3pBp6eb1goV6IkNie](url)   is the link to all mods

**Comment by covers1624** [comment]
Im not clicking on some random link for a list of mods, A FML log has all loaded mods listed.

**Comment by andrewt1738** [comment]
Call me stupid but where is the Fml log?

**Comment by shadoxxhd** [comment]
in the minecraft/logs folder

**Comment by sintaxcz** [comment]
i have the 1.7.10 modpack (http://www.technicpack.net/modpack/the-1710-pack.453902)
 + draconic evolution and this zoom bug happen too

**Comment by GreatOrator** [comment]
At this stage he has moved on to 1.9-1.10 so this may be a bug that remains.

**Comment by ThornsOfire** [comment]
I have had the same issue running Crackpack 1.1.0 on Minecraft 1.7.10 with fastcraft enabled. Using default mods on the FTB launcher. Can provide a complete list if necessary!

**Comment by brandon3055** [comment]
I will add this to my list of issues to look at if i ever push a new 1.7 build

**Comment by domi1819** [comment]
Numina (from Modular PowerSuits) is counteracting the FOV increase you get from being fast. Draconic Evolution also seems to do that, causing the final FOV value to be *much* lower that it should be.

**Comment by Philipp766** [comment]
I know this is a very late answer but I just ran into the same Problem. I looked through the keybind menu and there is a "Toggle Numia FOV Fix" if you have numia installed. This option is ON by default and seems to be not working right with Draconic/Modular Powersuits. Disabling this feature by assining a temporary key and toggling it fixes it for me.
I hope anyone who plays a old modpack once in a while will find that and itll save their day :)
